docs-ui/components/layout-thirds.stories.js

```diff
@@ -163,3 +206,26 @@ const Container = styled('div')`
 const MarginedButtonBar = styled(ButtonBar)`
   margin-bottom: ${space(1)};
 `;
+
+const StyledNavTabs = styled(NavTabs)`
```

Comment:
Is this something you'd need in application code as well? Should it be an export of the layout module?